### PR TITLE
Sync with develop

### PR DESCRIPTION
**Pull request by derobins**
Cherry pick of

ea7dfcd (Change Powershell to PowerShell in docs)

to

4497feb (Update H5Dget_space_status bug note to reference 1.14.0)

### CODE REVIEW

File: CMakePresets.json [hunk 1]
```diff
@@ -0,0 +1,237 @@
...
+        "LIBAEC_TGZ_ORIGNAME": {"type": "STRING", "value": "libaec-1.0.6.tar.gz"},
+        "LIBAEC_USE_LOCALCONTENT": "OFF"
+      }
+    },
+    {
+      "name": "ci-base-plugins",
+      "hidden": true,
+      "inherits": "ci-base-tgz",
+      "cacheVariables": {
+        "PLUGIN_TGZ_NAME": {"type": "STRING", "value": "hdf5_plugins-1.14.0.tar.gz"},
```
Comment (jhendersonHDF) [inline CMakePresets.json:41]: @byrnHDF Does the plugins repo version get updated with each release? Is there a plugins-1.14.1?

File: src/H5Dpublic.h [hunk 2]
```diff
@@ -461,6 +461,20 @@ H5_DLL hid_t  H5Dget_space_async(hid_t dset_id, hid_t es_id);
...
+ *       H5Dget_space_status() calculated the space allocation status by
+ *       comparing the sum of the sizes of all the allocated chunks in the
+ *       dataset against the total data size of the dataset, as calculated by
+ *       the number of elements in the dataset's dataspace multiplied by the
+ *       dataset's datatype size. If the dataset had any compression filters
+ *       applied to it and the dataset chunks were successfully compressed,
+ *       the sum of the sizes of the allocated dataset chunks would generally
+ *       always be less than the total data size of the dataset, and
+ *       H5Dget_space_status() wouldn't ever return
+ *       `H5D_SPACE_STATUS_ALLOCATED`.
```
Comment (jhendersonHDF) [inline src/H5Dpublic.h:476]: Should I close my 1.14 merge PR for this?

File: CMakePresets.json (hunk 1 above)
Comment (derobins) [reply CMakePresets.json:41]: If we're just targeting the filter API, the maintenance branch (e.g., 1.14) should be enough.

File: src/H5Dpublic.h (hunk 2 above)
Comment (derobins) [reply src/H5Dpublic.h:476]: Sure. Sorry - I just did a bulk pick of everything :/

Comment (jhendersonHDF) [reply src/H5Dpublic.h:476]: Ok I think you've captured everything in my 1.14 PRs so I'll close those.

**Review by jhendersonHDF — APPROVED** [review]

**Pull request merged**